src/main/java/io/jenkins/plugins/models/PluginRelease.java

```diff
@@ -33,26 +32,37 @@
   private String body;
 
   @JsonProperty("bodyHTML") public String getBodyHTML() {
-    MutableDataSet options = new MutableDataSet();
+    String baseUrl = "";
 
-    options.set(Parser.EXTENSIONS, Arrays.asList(
-      TablesExtension.create(),
-      StrikethroughExtension.create(),
-      GfmUsersExtension.create(),
-      TaskListExtension.create(),
-      GfmIssuesExtension.create(),
-      EmojiExtension.create()
-    ));
-    options.set(EmojiExtension.USE_SHORTCUT_TYPE, EmojiShortcutType.GITHUB);
     try {
-      options.set(GfmIssuesExtension.GIT_HUB_ISSUES_URL_ROOT, new URI(this.htmlUrl + "/../../../issues").normalize().toString());
+      baseUrl = new URI(this.htmlUrl + "/../../../issues").normalize().toString();
     } catch (URISyntaxException e) {
-      logger.error("Unable to process html_url", e);
+      e.printStackTrace();
     }
 
-    HtmlRenderer htmlRenderer = HtmlRenderer.builder(options).escapeHtml(true).build();
-    Parser markdownParser = Parser.builder(options).build();
-    return htmlRenderer.render(markdownParser.parse(this.body.replaceAll("<!--.*?-->", "")));
+    List<org.commonmark.Extension> extensions = Arrays.asList(
+        TablesExtension.create(),
+        AutolinkExtension.create(),
+        StrikethroughExtension.create(),
+        InsExtension.create()
+    );
+    HtmlRenderer htmlRenderer = HtmlRenderer.builder().extensions(extensions).escapeHtml(true).sanitizeUrls(true).build();
+    Parser markdownParser = Parser.builder().extensions(extensions).build();
+
+    Node document = markdownParser.parse(
+        EmojiParser.parseToUnicode(
+          this.body
+            .replaceAll("<!--.*?-->", "")
+            .replaceAll("#\\b([0-9]+)\\b", "[#$1](" + baseUrl + "/$1)")
+            .replaceAll("(\\b|\\s)+@(\\w+)(\\b|\\s)+", "$1[**@$2**](https://github.com/$2)$3")
+        )
+    );
+    return htmlRenderer
+      .render(document)
+      .replaceAll("\\s+rel=\"nofollow\"\\s+", " ")
```

Comment:
why?

Reply:
I was originally trying to keep my tests the same. (Because in true test driven development your supposed to update your code or tests not both)

Your right, I shouldn't be lazy and keep it in. I'll update it tonight